x-pack/plugins/cloud_security_posture/public/common/utils/get_primary_rule_tags.ts

```diff
@@ -5,15 +5,14 @@
  * 2.0.
  */
 
-const regexForCISSemanticVersioning = /CIS (0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)/;
+const regexForCISSemanticVersioning = /CIS ([0-9])(?:\.[0-9]\.?)?(?:[0-9])?/;
```

Comment:
1. not sure CIS follows semantic version. do we have reference for this?
2. tried a couple variations, went with the above, which satisfies the [these cases](https://regex101.com/r/IxQfVL/1):
```
// PASS
CIS 1.1.2
CIS 1.2 
CIS 1

// FAIL
CIS 1..2 
CIS 1.
```
3. i think we shouldn't look for known values in the `tags` array at all and should probably discuss alternatives